### PR TITLE
Add googlepubsub connector configuration

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Setting up the Google Pub Sub Environment  
+
+The Google Pub/Sub connector allows you to access the [Google Cloud Pub/Sub API Version v1](https://cloud.google.com/pubsub/docs/reference/rest/) through WSO2 EI.
+
+To work with the Google Pub/Sub connector, you need to have a Google Cloud Platform account. If you do not have a Google Cloud Platform account, go to [console.cloud.google.com](https://console.cloud.google.com/freetrial), and create a Google Cloud Platform trial account.
+
+Google Pub/Sub uses the OAuth 2.0 protocol for authentication and authorization. All requests to the Google Cloud Pub/Sub API must be authorized by an authenticated user. For information on how to obtain authentication and authorization user credentials.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:7]:
```suggestion
Google Pub/Sub uses the OAuth 2.0 protocol for authentication and authorization. All requests to the Google Cloud Pub/Sub API must be authorized by an authenticated user. For information on how to obtain authentication and authorization user credentials, see the following section.
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### Obtaining user credentials 
+
+Please follow the steps bellow for generate user credentials.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:11]:
```suggestion
Follow the steps below to generate user credentials.
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+**Follow the steps below to obtain a client ID and client secret**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:13]:
```suggestion
**Obtaining a client ID and client secret**
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+7. Search  **Google Cloud Pub/Sub API** under the **Big data or Networking category**.
+  
+    <img src="../../../../assets/img/connectors/pubsub-select-API.png" title="Pubsub API" width="600" alt="Pubsub API"/> 
+
+8. Click **Enable**. This enables the Google Cloud Pub/Sub API.   
+   
+    <img src="../../../../assets/img/connectors/pubsub-enable-api.png" title="Pubsub enable API" width="600" alt="Pubsub enable API"/> 
+
+**Follow the steps below to obtain an access token and refresh token**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:47]:
```suggestion
**Obtaining an access token and refresh token**
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Navigate to [oauthplayground](https://developers.google.com/oauthplayground), click the **gear icon** on the top right corner of the screen, and select Use your own **OAuth credentials**.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:49]:
```suggestion
1. Navigate to [OAuth playground](https://developers.google.com/oauthplayground), click the **gear icon** on the top right corner of the screen, and select **Use your own OAuth credentials**.
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+Please follow the steps bellow for generate user credentials.
+
+**Follow the steps below to obtain a client ID and client secret**
+
+1. Go to [https://console.developers.google.com/projectselector/apis/credentials](https://console.developers.google.com/apis/credentials), and sign in to your **Google account**.
+    
+    <img src="../../../../assets/img/connectors/pubsub-credentials-page.png" title="Google pubsub-credentials-page" width="600" alt="Google pubsub-credentials-page"/>   
+   
+2. If you do not already have a project, **create a new project**, click **Create credentials** and then select **OAuth client ID**.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:19]:
```suggestion
2. If you do not already have a project, you create a new project, click **Create credentials** and then select **OAuth client ID**.     
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+   
+3. Next, **select** Web Application, and **create a client**.
+  
+    <img src="../../../../assets/img/connectors/pubsub-select-web-application.png" title="Select web application" width="600" alt="Select web application"/> 
+
+4. Add [https://developers.google.com/oauthplayground](https://developers.google.com/oauthplayground/) as the redirect URL under **Authorized redirect URIs**, and then click **Create**. This displays the **client ID** and **client secret**.
+
+    <img src="../../../../assets/img/connectors/pubsub-authorization-redirect-URI.png" title="Authorization-redirect-URI" width="600" alt="Authorization-redirect-URI"/> 
+   
+5. Please a note the the **client ID** and **client secret** that is displayed, and then **click OK**.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:31]:
```suggestion
5. Note down the **client ID** and **client secret** that is displayed, and then **click OK**.   
```

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:7]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:11]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:13]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:47]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:49]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 6 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:19]: Updated.

File: en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md (hunk 7 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md:31]: Updated.

**Pull request merged**